tokio-test/src/io.rs

```diff
@@ -67,7 +67,7 @@ enum Action {
 struct Inner {
     actions: VecDeque<Action>,
     waiting: Option<Instant>,
-    sleep: Option<Sleep>,
+    sleep: Option<Pin<Box<Sleep>>>,
     read_wait: Option<Waker>,
     rx: mpsc::UnboundedReceiver<Action>,
```

Comment:
Note that this will probably merge conflict with [the stream PR](https://github.com/tokio-rs/tokio/pull/3277).

Reply:
Yeah, I will hold off a bit to see if we can get that one landed first.